README.md

```diff
@@ -27,6 +27,33 @@ Not to be confused with [`near-contract-standards`](https://crates.io/crates/nea
 
 You can think of this collection of common tools and patterns (mostly in the form of [derive macros](https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros)) as sort of an OpenZeppelin for NEAR.
 
+## Getting Started
+
+```
+rustup target add wasm32-unknown-unknown
+cargo init
+cargo add near-contract-tools
+cargo add near-sdk
+# https://raen.dev/guide/intro/getting-set-up.html
+cargo install raen
```

Comment:
Is RAEN an official, NF-sanctioned tool?

Reply:
If by "official, NF-sanctioned" you mean "owned and controlled by Pagoda or NF", then no, not yet. The RAEN team is a couple people who worked for NEAR Inc and have built RAEN funded by a grant (I think from NF) and are looking to fold it into Pagoda's tools at some point.

I find RAEN easier and more powerful than Pagoda's way.

Either way, though, I'll want to add more description to the Readme than the minimal lines I added here.